src/settings-ui/Settings.UI/SettingsXAML/Views/EnvironmentVariablesPage.xaml

```diff
@@ -2,51 +2,47 @@
     x:Class="Microsoft.PowerToys.Settings.UI.Views.EnvironmentVariablesPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls"
-    xmlns:controls1="using:CommunityToolkit.WinUI.Controls"
+    xmlns:custom="using:Microsoft.PowerToys.Settings.UI.Controls"
```

Comment:
@Jay-o-Way Why is this namespace `custom` now? That's rather confusing actually, since they are using the `.Controls` namespace? That's why we use `converters` namespace for Converters

Can't we use `controls` for PowerToys controls, and `toolkit` or `toolkitcontrols` for all controls part of the Toolkit?

Reply:
Actually, looking at the updated XAML.. `toolkitcontrols` is pretty verbose and makes the XAML harder to read :(.

Could we use `tk:SettingsCard` instead maybe? Or we do use `custom` for custom controls (it's used only in a handful of files) and use `controls` for Toolkit controls?

Reply:
> Why is this namespace `custom` now?

Because... In the `main` branch, `custom` is already (consistently) used through-out the OOBE xaml pages. Under the rest of _SettingsXAML_ area/folder, `controls`, `local`, and `custom` are all used. Just let me know which one is the best and I will apply to all of them. *
![image](https://github.com/microsoft/PowerToys/assets/65828559/0b363366-a0e8-4417-b1d5-14095851c03f)

https://github.com/microsoft/PowerToys/issues/18866#issuecomment-1806326170

> looking at the updated XAML.. `toolkitcontrols` is pretty verbose and makes the XAML harder to read :(.
> 
> Could we use `tk:SettingsCard` instead maybe?

Sure, that's possible. I would suggest `tkc` though, to include the "C" for controls. \* In that light: `ptc` for PowerToys Controls.

Reply:
> Why is this namespace `custom` now?

and

> looking at the updated XAML.. `toolkitcontrols` is pretty verbose and makes the XAML harder to read :(.
> 
> Could we use `tk:SettingsCard` instead maybe?In the `main` branch, `custom` is already (consistently) used through-out the OOBE xaml pages. Under the rest of _SettingsXAML_ area/folder, `controls`, `local`, and `custom` are all used.

@niels9001 Both **PowerToys** and **ToolKit** can have namespaces for **Controls** and **Converters** declared. That's four combinations. Simply using "controls" or "converters" is ambiguous. We can shorten "toolkit" to "tk" and "PowerToys" to "pt", but when both versions of **Controls** or **Converters** are used in the same file, we can't shorten it any more than that. (Aside from skipping vowels...)

Therefore I suggest

- xmlns:ptcontrols="using:Microsoft.PowerToys.Settings.UI.Controls"
- xmlns:ptconverters="using:Microsoft.PowerToys.Settings.UI.Converters"
- xmlns:tkcontrols="using:CommunityToolkit.WinUI.UI.Controls"
- xmlns:tkconverters="using:CommunityToolkit.WinUI.Converters"

https://github.com/microsoft/PowerToys/issues/18866#issuecomment-1806326170

Images:

![image](https://github.com/microsoft/PowerToys/assets/65828559/0b363366-a0e8-4417-b1d5-14095851c03f)
![image](https://github.com/microsoft/PowerToys/assets/65828559/96f3e861-e177-4dd0-b583-3cf3428a378f)
![image](https://github.com/microsoft/PowerToys/assets/65828559/8f8243ab-4842-4331-8e9e-8aaef28ba87a)